shell/browser/ui/win/notify_icon.cc

```diff
@@ -49,7 +49,11 @@ NotifyIcon::NotifyIcon(NotifyIconHost* host,
                        HWND window,
                        UINT message,
                        GUID guid)
-    : host_(host), icon_id_(id), window_(window), message_id_(message) {
+    : host_(host),
+      icon_id_(id),
+      window_(window),
+      message_id_(message),
+      weak_factory_(this) {
```

Comment:
The modern style for initializing the weak factory is to use a default-initializer (see below)

```suggestion
      message_id_(message) {
```